app/main.py

```diff
@@ -72,6 +72,11 @@ class form_in(BaseModel):
 
 class check(BaseModel):
     tweet_id: str
+	
+	
+class check_action(BaseModel):
```

Comment:
I can see where you added the new base model for this check_action.

Reply:
Good to check the base model. Approved